grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -101,16 +106,40 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
         actualDeliveryDate(nullable: true)
     }
 
+    String getUnitOfMeasure() {
+        if (quantityUom) {
+            return "${quantityUom?.code}/${quantityPerUom as Integer}"
+        }
+        else {
+            def g = ApplicationHolder.application.mainContext.getBean( 'org.codehaus.groovy.grails.plugins.web.taglib.ApplicationTagLib' )
```

Comment:
This might not be a good idea (performance-wise) but I'll see what it looks like with 100s of line items and make a decision later.